app/views/pages/get-started/build-your-first-app/sending-email-notifications.liquid

```diff
@@ -0,0 +1,215 @@
+
+If you are following this tutorial and building the _To Do_ app, then you probably would like to send the email after a new item has been added to the list.
+
+Since we have a separate page that handles this (`app\views\pages\item\create.liquid`), we can just plug into the right place and trigger the GraphQL mutation after the new item was successfully saved in the database:
+
+#### /app/views/pages/item/create.liquid
+
+{% raw %}
+```liquid
+---
+method: post
+---
+
+
+{% liquid
+	graphql result = 'item/create', title: context.params.title
+
+	if result.record.id
++		graphql email = 'item/email/created', data: context.params
+		redirect_to '/'
+	else
+		echo 'Something went wrong'
+		echo result.errors
+	endif
+%}
+```
+{% endraw %}
+
+In the 10th line we are triggering the GraphQL created for sending emails, but only if the record was added successfully.
+
+We are also passing the whole `context.params` hash as our `data` variable. This way every `<form>` element value will be passed to the GraphQL mutation, including the `title` that we are sending in the email.
+
+<img src="{{ 'images/get-started/todo/email_basic.jpg' | asset_url }}" width="742" height="346">
+
+<p>When we have everything configured and ready, you should see an email sent to the address you’ve provided in the configuration on Partners Portal.</p>
+
+
+## Sending emails to addresses provided dynamically
+
+In the `app/emails/item/created.liquid` we’ve hardcoded the `to:` address, but in real life you will probably want to insert any address passed as a variable. For example when you’ll implement any form of user accounts and would like to send the emails to the owner of that account.
+
+{% include 'alert/tip', content: 'Please remember that on the free <em>Staging</em> environment emails can be sent only for the test email address. To send emails to other addresses you will have to sign up for the <em>Production</em> environment.' %}
+
+It is totally possible to use a variable in the configuration section of a page. As an example, in `app/emails/item/created.liquid` we could use `{% raw %}{{ data.recipient }}{% endraw %}`:
+
+#### /app/emails/item/created.liquid
+
+{% raw %}
+```yaml
+---
+from: "no-reply@example.com"
+to: "{{ data.recipient }}"
+subject: "New item on the To Do list"
+layout: mailer
+---
+```
+{% endraw %}
+
+Notice that the Liquid variable is surrounded with quotation marks.
+
+This would also mean that we need to pass the `recipient` variable to our GraphQL mutation. Since we are passing the whole `context.params` as described above, we could just simply use a hidden form element in the `<form>`:
+
+#### /app/views/pages/index.liquid
+
+```html
+<!-- form for adding new item -->
+<form action="/item/create" method="post">
++ <input type="hidden" name="recipient" value="recipient-email-address@example.com">
```

Comment:
I've noted it in one of the previous PR that we lack CSRF token here and it will cause problems in real-life application if only registered users will be able to add a new item - noting it again here :)

Reply:
True, but I would like to avoid 'put it here, it will come in handy later' in the beginners tutorial (that this is the last part of I think) and instead do a separate article about how to handle user management and then introduce the CSRF token on a living example.